### DISCUSSION TITLE
Error: Error occurred while verifying paramsencoding/hex: invalid byte: U+0079 'y

### DISCUSSION THREAD

**Discussion by salim-mahat** [Web3Auth/Web3Auth#1305] [Help]
I've been using the Web3Auth Node.js SDK to implement onboarding a user with the Web3Auth feature, and I've been following the instructions at https://web3auth.io/docs/sdk/node/, but I'm stuck at the connect method and getting an error. Error: While checking paramsencoding/hex, an error occurred: incorrect byte: 'y' U+0079

my code 
`const { Web3Auth } = require("@web3auth/node-sdk");
const web3auth = new Web3Auth({
clientId: "BKvkul2KnF0N55JFjQJZ7jrENfZjhsoJ6wbh_iOxi93G3yRCv56vrxWfa4QAZ1cqIre6xTxG7vZIOOr8ewHS-tA", // Get your Client ID from Web3Auth Dashboard
  chainConfig: {
    chainNamespace: "eip155",
    chainId: "0x13881",
    rpcTarget: "https://rpc-mumbai.maticvigil.com/", // needed for non-other chains
  },
});
web3auth.init({ network: "testnet" });
const connect = async () => {
const provider = await web3auth.connect({
  verifier: "torus", // replace with your verifier name, and it has to be on the same network passed in init().
  verifierId: "venuthenftbrewery", // replace with your verifier id(sub or email), setup while creating the verifier on Web3Auth's Dashboard
  idToken: "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjFiYjk2MDVjMzZlMTAyMDMwNjkzODY4MzAyMDJiMmQifQ.eyJzdWIiOiJzYWxpbV9tYWhhdF84IiwibmFtZSI6InNhbGltIG1haGF0IiwiZW1haWwiOiJzYWxpbS5tYWhhdEBrbWluZHouaW4iLCJlbWFpbF92ZXJpZmllZCI6dHJ1ZSwiaWF0IjoxNjc1OTQxNDgyLCJleHAiOjE2NzU5NDUwODJ9.-RcCltHKggSMmXpHGLRM8XTARU379YAQ8JLd1XMF4MxuySflDLcFJ5mP95TKV3K66uEySF70yXOULc_Xo5KCCg", // replace with your newly created unused JWT Token.
});
  console.log("Provider -->", provider);
const privateKey = await provider.request({ method: "eth_private_key" });
console.log("ETH Private Key", privateKey);

};
connect();`

**Comment by ieow** [comment]
Try change your `verifierId`'s value with `"sub"` or `"email"`

**Reply by salim-mahat** [reply]
I adjusted this, but the problem persists.
